vyxal/elements.py

```diff
@@ -2854,9 +2856,24 @@ def regex_sub(lhs, rhs, other, ctx):
 def remove(lhs, rhs, ctx):
     """Element o
     (any, any) -> a.remove(b)
+    (fun, num) -> first b positive integers where a is truthy
```

Comment:
This doesn't have the (num, fun) overload. Also, could the (any, any) overload be moved to the bottom here too?